test/docfx.Test/lsp/LanguageServerTestCommand.cs

```diff
@@ -12,8 +12,14 @@ public class LanguageServerTestCommand
 
         public Dictionary<string, string> EditFiles { get; init; }
 
+        public Dictionary<string, string> CreateFiles { get; init; }
+
+        public Dictionary<string, string> ChangeFilesWithoutEditor { get; init; }
```

Comment:
nit: to align with `EditFiles`
```suggestion
        public Dictionary<string, string> EditFilesOutsideEditor { get; init; }
```